resources/gherkin-languages.json

```diff
@@ -26,7 +26,8 @@
     "name": "Afrikaans",
     "native": "Afrikaans",
     "rule": [
-      "Regel"
+      "Reël",
+      "Reel"
```

Comment:
Although this **removes** a keyword for "rule" that was previously supported, it does not have any impact on end-users because we do not currently parse / load / support the `Rule` keyword anyway.